requirements.txt

```diff
@@ -7,10 +7,8 @@ amazon-braket-schemas==1.19.1
 amazon-braket-sdk==1.56.0
 amazon-braket-algorithm-library==1.4.1
 cvxpy==1.3.1
-dask==2023.1.1
 ecos==2.0.12
 ipykernel==6.21.1
-jax==0.4.2
```

Comment:
Can you confirm about jax? I think it is used in one of the notebooks @maolinml has been recently working on.

Reply:
I did a pip freeze on the integ-tests installs as well as looked through our examples. I did not see it being actively used.

Reply:
Okay, I was mentioning about this [PR](https://github.com/amazon-braket/amazon-braket-examples/pull/337/files) which introduces the jax requirement.